.github/workflows/update-dependencies.yml

```diff
@@ -2,7 +2,7 @@ name: "Update Dependencies"
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 0 * * FRI'
+    - cron: '0 */6 * * *'
```

Comment:
Why not? :)

Reply:
I have GH set to notify about all activities and this would be pretty spammy